### DISCUSSION TITLE
Reference for measured capacity and Capacity [A.h]

### DISCUSSION THREAD

**Discussion by manjunathnilugal** [pybamm-team/PyBaMM#2501] [Q&A] [answered]
How Measured capacity [A.h] and Capacity [A.h] in pybamm summary variables are calculated ?

**Comment by RuiheLi** [comment, marked as answer]
Hi~ the easiest way to find the definition of some variables if to search the PyBaMM in some code editor, such as [VS CODE](https://code.visualstudio.com/). See below:
![image](https://user-images.githubusercontent.com/84007676/204485000-d4e9f4cc-d4ae-47d7-9815-5919e867ccec.png)

You can then click into the source file and check extactly how it comes from.
